vendingMachine.js

```diff
@@ -1,68 +1,90 @@
 const prompt = require('prompt-sync')();
-let products = [
+const productsMachine = [
   ["Chocorramo", "Gansito", "Galletas"],
   ["Maizitos", "Tostacos", "Yogurt"],
   ["Papas", "Platanitos", "Ponque"]
 ];
-let stock = [
+const productStock = [
   [5, 5, 5],
   [5, 5, 5],
   [5, 5, 5]
 ];
 
 function showVendingMachine(arr1, arr2) {
-  let result = '';
-  let coord, stoc;
+  const space = " ";
+  const supline = "-";
+  const topLine = supline.repeat(122);
+  const spaceTable = 40; //40 size of spaces for word container in table
+  let itemTable = '';
+  let coord, stock, spaceLeft1, spaceLeft2, spaceRight1, spaceRight2;
+  console.log(topLine);
   for (let i = 0; i < arr1.length; i++) { //0
     for (let j = 0; j < arr1[i].length; j++) { //2
-      result = result + arr1[i][j] + '    ';
+      let a = 0;
+      let left, rigth, squareItem;
+      left = space.repeat(Math.round((spaceTable - (arr1[i][j].length)) / 2));
+      //** residual operator (%)*/
+      //** Returns the integer value resulting from the division of the operands, known as the remainder. */
+      //** In this case it is used to know the length of the array content in that coordinate, and to know if it is odd or even. */
+      if (arr1[i][j].length % 2 === 1) {
+        a = 1;
+      }
+      rigth = space.repeat(Math.round((spaceTable - (arr1[i][j].length)) / 2) - a);

```

Comment:
Right, what? spaces? dashes?

Reply:
residual operator (%)
 Returns the integer value resulting from the division of the operands, known as the remainder.
 In this case it is used to know the length of the array content in that coordinate, and to know if it is odd or even.

Reply:
Done